src/include/miopen/mlo_internal.hpp

```diff
@@ -167,6 +167,9 @@ size_t setBotDfDescFromMLDesc(int spatial_dims, TTo& to, const TensorDescriptor&
     return SetDescFromMLDesc(spatial_dims, to, tensor, &TTo::setBotDfDescr);
 }
 
+MIOPEN_INTERNALS_EXPORT auto MakeConvDbGetter(const ExecutionContext& ctx)
+    -> std::function<PerformanceDb&()>;
+
```

Comment:
[R] I think we shouldn't add new things into this file. This file has confusing name, also it probably has a license problem.